fineract-provider/src/main/resources/logback.xml

```diff
@@ -44,6 +44,7 @@
 	<logger name="org.springframework.beans" level="info" />
 	<logger name="net.sf.ehcache" level="error" />
 	<logger name="org.hibernate.cache" level="error" />
+	<logger name="com.zaxxer.hikari" level="debug" additivity="false" />
```

Comment:
I'm wondering, won't this (at level debug instead of maybe warn) create a lot of extra logging? 

I don't mind merging it as is (I'm just re-reading everything else), we can always change this log level later.